Monika After Story/game/pong.rpy

```diff
@@ -731,7 +732,7 @@ label mas_pong_dlg_winner:
 
     #The player has lost 3, 8, 13, ... matches in a row.
     elif loss_streak_counter >= 3 and loss_streak_counter % 5 == 3:
-        m 2eka "Come on, [player], I know you can beat me..."
+        m 2eka "You can do this, [player]..."
```

Comment:
"You can do this" doesn't really seem to sound good with an ellipsis at the end, I believe.